vendor/github.com/aosedge/aos_common/api/cloudprotocol/unitstatus.go

```diff
@@ -171,13 +171,13 @@ type ComponentStatus struct {
 type UnitStatus struct {
 	MessageType  string             `json:"messageType"`
 	IsDeltaInfo  bool               `json:"isDeltaInfo"`
-	UnitConfig   []UnitConfigStatus `json:"unitConfig"`
-	Nodes        []NodeInfo         `json:"nodes"`
-	Services     []ServiceStatus    `json:"services"`
-	Instances    []InstanceStatus   `json:"instances"`
+	UnitConfig   []UnitConfigStatus `json:"unitConfig,omitempty"`
```

Comment:
does it mean that we don't provide empty array at all to the cloud or it omits a nil value only?